resources/cloudtrail_trails.go

```diff
@@ -248,14 +248,13 @@ func fetchCloudtrailTrails(ctx context.Context, meta schema.ClientMeta, parent *
 			}
 			for _, tr := range trails {
 				tags := getCloudTrailTagsByResourceID(*tr.TrailARN, response.ResourceTagList)
-				if len(tags) == 0 {
-					continue
-				}
-
 				wrapper := CloudTrailWrapper{
 					Trail: tr,
 					Tags:  make(map[string]interface{}, len(tags)),
 				}
+				if len(tags) == 0 {
+					res <- wrapper
```

Comment:
isn't it sent twice now?

Reply:
whole block (255-257) needs to be removed